amida-tech/indaba-client
# Changes for the demo login page.

#### What's this PR do?
Changes some of the language for the login page.

#### Related JIRA tickets:
INBA-267.

#### How should this be manually tested?
Go to `http://localhost:3000/login`
Confirm the username field reads "Username or email"
Confirm that the field below reads "Forgot username?"

#### Any background context you want to provide?
#### Screenshots (if appropriate):
